server/models/timeManager.js

```diff
@@ -0,0 +1,112 @@
+const { Time } = require("./timeModel");
+const { WeekPhase } = require("./weekPhase");
+var cron = require('node-cron');
+const dayjs = require("dayjs");
+
+
+//Handle of phase called only when phase is enabled
+
+class TimeManager {
+    constructor(){
+        this.phaseList = this.getPhaseList();
+        this.activePhase = this.getTrueActivePhase();
+        this.activePhase.handler();
```

Comment:
let's decide the cron if it's time to run the handler. This can lead to error (and I'm pretty sure is a big bug)

Reply:
If this is about the comment, I think that comment is not written well. The handler function is ONLY called inside the function we pass to cron as parameter.

Reply:
Yep, I saw the condition in the cron method and it should work.